src/agent/debuglet.ts

```diff
@@ -205,6 +205,15 @@ export class Debuglet extends EventEmitter {
    * @private
    */
   async start(): Promise<void> {
+    process.on('warning', (warning) => {
+      if ((warning as any).code ===
+          'INSPECTOR_ASYNC_STACK_TRACES_NOT_AVAILABLE') {
+        console.log(
+            'The current debug agent does not use Inspector async stack ' +
+            'traces. The above warning will not affect the debug agent.');
```

Comment:
I think we should say use `Stackdriver Debug Agent` (or something similar) instead of `debug agent` to make it clear that it is specifically the Stackdriver agent that is not impacted.

Reply:
Done. see above changed message.